Datagrid/ListMapper.php

```diff
@@ -124,6 +124,34 @@ public function add($name, $type = null, array $fieldDescriptionOptions = array(
             );
         }
 
+        // NEXT_MAJOR: Remove first "if" statement (when requirement of Symfony is >= 2.8)
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) { // symfony >= 2.8
+            $choices = $fieldDescription->getOption('choices');
+            if (($type == 'choice' || $type == 'Symfony\Component\Form\Extension\Core\Type\ChoiceType') && $choices) {
+                // NEXT_MAJOR: Remove, leave only code from "else" (when requirement of Symfony is >= 3.0)
+                if (method_exists('Symfony\Component\Form\FormTypeInterface', 'setDefaultOptions')) { // 2.8 <= symfony < 3.0
```

Comment:
The comment you write here is the same as the comment line 128, written in the other direction. Same meaning though.